### ISSUE TITLE
feature request: jumping between buffers

### ISSUE THREAD

**Issue by luminalle** [dosimple/workspace.vim#10]
hello,

i kinda love this plugin, well done.

One thing I'm missing is the ability to use something like :b{number} to jump between buffers, but only in current workspace. How and why? Because I ignore the real buffer numbers and use relative numbers inside workspace, meaning that buffer number 1 just means the first buffer in :ls. I have the relative numbers visible in my tabline already, using https://github.com/bagrat/vim-buffet . I know this might not make sense to many users, but I think this setup would be awesome.

tldr; need a :b1/:b2/:bn like feature that works like :ls, :bn, and :bp in this plugin

There is also one other feature I want to mention that is not that important to me, but maybe it would make sense to make workspaces to stick to current directory context. So that when :cd is called from workspace1, it would not change the directory of workspace2.

This is maybe the most undervalued vim plugin. It's like having ten instances of vim open, but within one vim.

**Comment by luminalle** [comment]
```
function! WS_GoBuf(relIndex) abort
    let tabNumber = gettabvar(tabpagenr(), "WS")
    let tabBuffers = WS_Buffers(tabNumber)
    if a:relIndex > len(tabBuffers)
        return
    elseif a:relIndex < 1
        return
    else
        let targetBuf = tabBuffers[a:relIndex-1].bufnr
        execute 'buffer' targetBuf
    endif
endfunction

nnoremap <A-1> :call WS_GoBuf(1)<CR>
nnoremap <A-2> :call WS_GoBuf(2)<CR>
nnoremap <A-3> :call WS_GoBuf(3)<CR>
nnoremap <A-4> :call WS_GoBuf(4)<CR>
nnoremap <A-5> :call WS_GoBuf(5)<CR>
nnoremap <A-6> :call WS_GoBuf(6)<CR>
nnoremap <A-7> :call WS_GoBuf(7)<CR>
nnoremap <A-8> :call WS_GoBuf(8)<CR>
nnoremap <A-9> :call WS_GoBuf(9)<CR>
nnoremap <A-0> :call WS_GoBuf(10)<CR>
```
This does the trick. I'm not too experienced in vim scripting, maybe there is better way to do the mapping.

just noticed that the tabline I use already implements this...
